server/lib/weather/weather.command.js

```diff
@@ -17,13 +55,104 @@ async function command(message, classification, context) {
     if (!house || !house.latitude || !house.longitude) {
       throw new NoValuesFoundError();
     }
+    const day = moment().subtract(
+      moment()
+        .toDate()
+        .getTimezoneOffset(),
+      'minutes',
+    );
+    let hour = 0;
+    let minute = 0;
     switch (classification.intent) {
       case 'weather.get':
         weather = await this.get(house);
         context.temperature = weather.temperature;
         context.units = weather.units === 'si' ? '°C' : '°F';
         await this.messageManager.replyByIntent(message, `weather.get.success.${weather.weather}`, context);
         break;
+      case 'weather.getPrevisions':
+        if (context.time) {
+          house.target = 'hourly';
```

Comment:
Pourquoi faire une mutation sur un paramètre de cette fonction? Cela n'a pas de sens que la maison est un attribut "target"?

Reply:
Parce que selon la requete, cette variable peut avoir 3 valeurs : currently, hourly and daily.
Et le retour de l'API est différent selon ces 3 uses cases.

Le nom target est peut être pas le plus pertinent, mais j'avoue manquer d'inspiration ^^

Reply:
En fait mon problème c'est surtout que tu fais une mutation sur l'objet "house", ça n'a pas de sens! Pourquoi une maison aurait un attribut target?